app/Http/Controllers/FoodController.php

```diff
@@ -37,6 +37,22 @@ public function index()
     }
 
     /**
+     * Show the list food by id and name if exists.
+     *
+     * @param \Illuminate\Http\Request $request request value
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function getById(Request $request)
```

Comment:
`getById` is mean ID of Food or category? change to `getByCategory` if condition is category